parsers/UA.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import arrow
+import dateutil
+import requests
+
+MAP_GENERATION = {
+  'aes': 'nuclear',
+  'tec': 'other', # fossil 
+  'tes': 'coal',
+  'chpp': 'gas',
+  'hpp': 'hydro',
+  'vde': 'wind',
+  'gesgaes' : 'gesgaes', # hydro pumped storage
+  'consumptiongaespump' :'consumptiongaespump' #hydro run of river and poundage
+}
+MAP_OTHER   = {
+  'consumption' : 'consumption',
+  'hour' : 'hour'
+}
+
+tz = 'Europe/Kiev'
+
+def fetch_production(country_code='UA', session=None):
+    r = session or requests.session()
+    
+    today =  arrow.now(tz=tz).format('DD.MM.YYYY')
+    url = 'https://ua.energy/wp-admin/admin-ajax.php'
+    postdata = {
+      'action': 'get_data_oes',
+      'report_date': arrow.now(tz=tz).format('DD.MM.YYYY'),
+      'type': 'day',
+      'rnd': 0.2046847583117225

```

Comment:
What does this correspond to? Seems like a "random" number (from the name `rnd`).